src/utils/constants.js

```diff
@@ -0,0 +1,5 @@
+export const BreakpointsEnum = Object.freeze({
```

Comment:
Started with the 3 most common breakpoints. This can easily be expanded or adjusted in the future

Reply:
Is there a reason for using a mix of min-width and max-widths? I generally try to stick with one or the other for clarity, but I could easily be missing something. Maybe it might be hard to explicitly define either the mobile or desktop breakpoints for this use case?

Reply:
I'm looking at it as more of defining a basis of the viewports for the project. 
- Desktop = sees all windows/sections
- Mobile might not see all (but only sees most important windows/sections). 
@willcary @will-johnston-goodyear

Reply:
It is a good practice to use a mobile-first approach to media queries, applying the least amount of rules on mobile for a better performance on low-end devices. In that case the we should only use `min-width`s and leverage the desktop machine's capability of applying more rules to adapt the mobile layout for their users.

BUT, since the project has started as a desktop-first, I think the breakpoints look ok. Perhaps changing the `TABLET` one to include a `min-width: 415px`, but that's another topic I guess.